packages/jest-cli/src/testSchedulerHelper.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
```suggestion
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
```

Reply:
done :) i see `babel` used to TS compiler with `noImplictAny` flag to track none typed